gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiResource.java

```diff
@@ -574,6 +575,11 @@ public Response reviewsReject(@Context HttpHeaders headers, @PathParam("apiId")
         return Response.noContent().tag(Long.toString(updatedApi.getUpdatedAt().getTime())).lastModified(updatedApi.getUpdatedAt()).build();
     }
 
+    @Path("/pages")
+    public ApiPagesResource getApiPagesResource() {
+        return resourceContext.getResource(ApiPagesResource.class);
+    }
+
```

Comment:
**question**: I think we have avoided this kind of inclusion in other resources. Is there any special reason to include the sub resource like this? If not, I would recommend to just add `@Path("/environments/{envId}/apis/{apiId}/pages")` directly in `ApiPagesResources`

Reply:
OK I was not aware of that. I will do the change